server/src/main/java/io/druid/metadata/SQLMetadataConnector.java

```diff
@@ -368,6 +368,22 @@ public void createLockTable(final String tableName, final String entryTypeName)
     );
   }
 
+  public void createOwnerTable(final String tableName, final String entryTypeName) {
+    createTable(
+        tableName,
+        ImmutableList.of(
+            String.format(
+                "CREATE TABLE %1$s (\n"
+                + "  %2$s_id VARCHAR(255) NOT NULL,\n"
+                + "  owner_id VARCHAR(255) NOT NULL,\n"
+                + "  PRIMARY KEY (%2$s_id)\n"
```

Comment:
Shouldn't primary key be composite of entryTypeName and owner_id, i. e. there is 1-to-1 correspondence?